utest/reporting/test_jsmodelbuilders.py

```diff
@@ -19,7 +19,10 @@
 
 def remap(model, strings):
     if isinstance(model, StringIndex):
-        return strings[model][1:]
+        if strings[model].startswith('*'):
+            # Strip the asterisk from a raw string.
+            return strings[model][1:]
+        return strings[model]
```

Comment:
Would it actually be enough to decompress the string here? It ought to be easier than compressing expected results. It would also be easier to analyse errors if values wouldn't be compressed.